dspace-server-webapp/src/test/java/org/dspace/app/rest/matcher/SubmissionFormFieldMatcher.java

```diff
@@ -113,19 +116,26 @@ private SubmissionFormFieldMatcher() {
      *            the optional filter to be used for the lookup
      * @param searchConfiguration
      *            the searchConfiguration to be used for the lookup
+     * @param nameVariants
+     *            the optional name variants allowed flag
      * @return a Matcher for all the condition above
      */
     public static Matcher<? super Object> matchFormOpenRelationshipFieldDefinition(String type, String label,
                                                                                    String mandatoryMessage,
-                                                                                   boolean repeatable, String hints,
+                                                                                   boolean repeatable,
+                                                                                   String hints,
                                                                                    String metadata,
                                                                                    String relationshipType,
                                                                                    String filter,
-                                                                                   String searchConfiguration) {
+                                                                                   String searchConfiguration,
+                                                                                   @Nullable Boolean nameVariants) {
         return allOf(
             hasJsonPath("$.selectableRelationship.relationshipType", is(relationshipType)),
             hasJsonPath("$.selectableRelationship.filter", is(filter)),
             hasJsonPath("$.selectableRelationship.searchConfiguration", is(searchConfiguration)),
+                nameVariants != null ?
+                        hasJsonPath("$.selectableRelationship.nameVariants", is(nameVariants.toString()))
```

Comment:
Since `nameVariants` is `@Nullable`, couldn't we end up with a NPE here if it is set to `null`?  This same issue also exists in `matchFormClosedRelationshipFieldDefinition` below

Reply:
This has been adjusted